### DISCUSSION TITLE
If I don't install any mods, is FabricMC exactly the same as vanilla?

### DISCUSSION THREAD

**Discussion by horizon86** [FabricMC/fabric-api#4204] [FAQ] [answered]
I’d like to know if FabricMC makes any changes to game mechanics, even minor ones (though not as extensive as those in Paper).

**Comment by Shnupbups** [comment, marked as answer]
There are two major components to Fabric - Fabric Loader, and Fabric API. Fabric Loader is the actual mod loader, and Fabric API is a separately installed mod which most other mods will require to function.

Fabric Loader makes absolutely no changes to Minecraft's gameplay, mechanics, etc. - so yes, it's exactly the same as vanilla.

Fabric _API_ however, attempts to make as few changes as possible but it cannot guarantee that everything will be exactly the same as vanilla.

So yes, without any mods at all, Fabric is the same as vanilla. With the Fabric API mod, it might not be.

**Reply by modmuss50** [reply]
Yes that's correct. Even with Fabric API there are no known changes, and any that are found and reported are resolved as a priority.
